source/Ship.cpp

```diff
@@ -3020,6 +3020,18 @@ double Ship::DisabledHull() const
 
 
 
+// Get the (absolute) amount of hull that needs to be damaged until the
+// ship becomes disabled, returns 0 if the ships hull is already below the
+// disabled level.
+double Ship::HullUntilDisabled() const
+{
+	// Need to overshoot a little bit, since a ship only gets disabled when
+	// we get _below_ the MinimumHull.
```

Comment:
Using a slightly tweaked comment from what I posted on Discord.
```suggestion
	// Ships become disabled when they surpass their minimum hull threshold,
	// not when they are directly on it, so account for this by adding a small amount
	// of hull above the current hull level.
```